pkg/controller/certificaterequests/selfsigned/selfsigned_test.go

```diff
@@ -99,7 +99,7 @@ func TestSign(t *testing.T) {
 
 	skEC, err := pki.GenerateECPrivateKey(256)
 	if err != nil {
-		t.Errorf("failed to generate ECDA private key: %s", err)
+		t.Errorf("failed to generate ECDSA private key: %s", err)
```

Comment:
notable. I'm not actually sure if this is the right thing. maybe it's just an EC ??

Reply:
I would prefer `EC`